### PR TITLE
Task/3794 update email styles

### PR DESCRIPTION
**Pull request by tiwariav**

### CODE REVIEW

**Comment by tiwariav** [comment]
@verysanghee can you go through the translation files to check for changes in translations as well as new required ones?

**Comment by verysanghee** [comment]
For a new VM mail, 
- the btn (show detail) goes to https://dynamicweb-development.ungleich.ch/. 
- the outline of the mail should be removed. all mails are without outline. 
- the logo does not load
screen shot 2017-09-23 at 14 47 07

**Comment by pcoder** [comment]
Also we seem to have an additional double quote in hosting/locale/de/LC_MESSAGES/django.po:293

File: datacenterlight/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -132,36 +132,42 @@ msgstr ""
...
+msgid ""
+"Thanks for joining us! We provide the most affordable virtual machines from "
+"the heart of Switzerland."
 msgstr ""
-"Klicke hier, um deinen Data Center Light Account zu aktivieren oder kopiere "
-"den folgenden Link in die Adressleiste deines Browsers.\n"
-"%(base_url)s%(activation_link)s\n"
+
+msgid "Try now, order a VM. VM price starts from only 15CHF per month."
+msgstr ""
```
Comment (verysanghee) [inline datacenterlight/locale/de/LC_MESSAGES/django.po]: "Unser Angebot beginnt bei 15 CHF pro Monat. Probier's jetzt aus!"

File: datacenterlight/locale/de/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -132,36 +132,42 @@ msgstr ""
...
+"browser to activate your Data Center Light account."
+msgstr "Kopiere den folgenden Link in die Adressleiste deines Browsers."
+
+msgid ""
+"You can copy and paste the following link into the address bar of your "
+"browser to activate your Data Center Light account."
+msgstr "Kopiere den folgenden Link in die Adressleiste deines Browsers."
+
+msgid "Welcome to Data Center Light!"
+msgstr "Willkommen zu Data Center Light!"
```
Comment (verysanghee) [inline datacenterlight/locale/de/LC_MESSAGES/django.po]: "Willkommen beim Data Center Light!"

File: datacenterlight/locale/de/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -132,36 +132,42 @@ msgstr ""
...
+
+msgid ""
+"Thanks for joining us! We provide the most affordable virtual machines from "
+"the heart of Switzerland."
 msgstr ""
```
Comment (verysanghee) [inline datacenterlight/locale/de/LC_MESSAGES/django.po]: "Bei uns findest Du die günstiges VMs aus der Schweiz."

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -189,101 +189,62 @@ msgid "Support / Contact"
...
-msgid ""
-"\n"
-"You're receiving this email because you requested a password reset for your "
-"user account at %(site_name)s.<br/>\n"
-"Please go to the following page and choose a new password: %(base_url)s"
-"%(password_reset_url)s<br/>\n"
-"If you didn't request a new password, ignore this e-mail.<br/>\n"
-"Thank you!\n"
+msgid "We received a request to reset your password."
 msgstr ""
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: "Wir haben eine Anfrage erhalten, um Dein Passwort zurückzusetzen."

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -189,101 +189,62 @@ msgid "Support / Contact"
...
-"Thank you!\n"
-msgstr ""
-"Du erhälst diese E-Mail da du dein Passwort für deinen Account bei "
-"%(site_name)s zurücksetzen möchtest.\n"
-"Bitte folge diesem Link und wähle ein neues Passwort: %(base_url)s"
-"%(password_reset_url)s Solltest du kein neues Passwort angefordert haben, "
-"dann ignoriere diese E-Mail.\n"
-"Dankeschön!\n"
+msgid "If you didn't make this request you can safely ignore this email."
+msgstr ""
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: "Falls Du kein neues Passwort angefragt hast, kannst Du diese E-mail ignorieren."

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -189,101 +189,62 @@ msgid "Support / Contact"
...
 
-#, python-format
-msgid ""
-"You're receiving this mail because your virtual machine [%(vm_name)s] has "
-"been cancelled.<br/>\n"
-"You can see your order status by clicking [my VM page] below.<br/>\n"
-"If you want to order a new virtual machine, you can do it by clicking <a "
-"href=\"%(base_url)s%(my_virtual_machines_url)s\">this link</a>.<br/>\n"
+msgid "Otherwise, click here to reset your password."
 msgstr ""
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: "Andernfalls klicke hier, um Dein Passwort zurückzusetzen."

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -189,101 +189,62 @@ msgid "Support / Contact"
...
-"Du erhälst diese E-Mail, da deine virtuelle Maschine [%(vm_name)s] gekündigt "
-"wurde.<br/>\n"
-"Um deinen Auftragsstatus zu sehen, klicke auf die [my VM page] unten.<br/>\n"
-"Falls du eine neue virtuelle Maschine bestellen möchtest, kannst du dies "
-"tun, indem du <a href=\"%(base_url)s%(my_virtual_machines_url)s\">diesen "
-"Link klickst</a>.<br/>\n"
 
-msgid "My VM page"
+msgid "Thank you!"
 msgstr ""
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: "Danke schön."

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -189,101 +189,62 @@ msgid "Support / Contact"
...
-"tun, indem du diesen Link klickst.\n"
-"%(base_url)s%(my_virtual_machines_url)s\n"
+"You are receiving this email because your virutal machine [%(vm_name)s] has "
+"been cancelled."
+msgstr ""
+"Du erhälst diese E-Mail, da deine virtuelle Maschine [%(vm_name)s] "
+"gekündigt wurde."
+
+msgid "You can always order a new VM by clicking the button below."
+msgstr ""
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: "Du kannst einfach eine neue VM bestellen, indem Du den Knopf weiter unten drückst."

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 9]
```diff
@@ -189,101 +189,62 @@ msgid "Support / Contact"
...
+
+msgid "CREATE VM"
+msgstr "NEUE VM"
+
+msgid "You can always order a new VM by following the link below."
+msgstr ""
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: ...this part looks repetitive. do we have a btn or do we have a link? @tiwariav

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 10]
```diff
@@ -189,101 +189,62 @@ msgid "Support / Contact"
...
 
 msgid "Toggle navigation"
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: Where do we have "toggle navigation"??

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 11]
```diff
@@ -646,8 +600,8 @@ msgid ""
 "Your VM will be up and running in a few moments. We will send you a "
 "confirmation email as soon as it is ready."
 msgstr ""
-"Deine VM ist gleich bereit. Wir senden Dir eine Bestätigungsemail, "
-"sobald Du auf sie zugreifen kannst."
+"Deine VM ist gleich bereit. Wir senden Dir eine Bestätigungsemail, sobald Du "
+"auf sie zugreifen kannst."
 
 msgid "In order to create a VM, you need to create/upload your SSH KEY first."
 msgstr ""
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: "Um eine VM zu erstellen musst Du zuerst einen SSH-Key erstellen / hochladen."

File: hosting/locale/de/LC_MESSAGES/django.po (hunk 10 above)
Comment (tiwariav) [reply hosting/locale/de/LC_MESSAGES/django.po]: its for screen readers for top navigation toggle button

File: hosting/locale/de/LC_MESSAGES/django.po (hunk 9 above)
Comment (tiwariav) [reply hosting/locale/de/LC_MESSAGES/django.po]: in html email, we have a button. in plain text email we cannot have a button,so we have a link.

**Comment by verysanghee** [comment]
the btn text("NEUE VM"): the left padding looks bigger than right padding..
signage "Dein Data Center Light Team" font-weight: to be 400 instead of 600.

**Comment by verysanghee** [comment]
pw reset while testing on this PR redirected to server 500 error

**Comment by verysanghee** [comment]
You have ordered a new virtual machine!
Your order of [ CentOS 7-19969 ] has been charged.
You can view your VM detail by clicking the button below.

from [VM number], [] should be removed. [] was their only for our reference.

**Pull request merged**